index.js

```diff
@@ -67,7 +67,13 @@ function PinoColada () {
     if (contentLength != null) output.push(formatBundleSize(contentLength))
     if (responseTime != null) output.push(formatLoadTime(responseTime))
 
-    return output.filter(noEmpty).join(' ')
+    var str = output.filter(noEmpty).join(' ')
+
+    // if error, output stack trace on new line
+    var err = obj.err
+    if (err && err.stack) str += nl + err.stack
```

Comment:
oh okay cool, i didn't realize that we were printing a stack trace for `fatal`. my errors are coming from [here in `pino-http`](https://github.com/pinojs/pino-http/blob/2778f077ab5b3390af7d01fc550c475ff7b1d925/logger.js#L35-L42), which calls `log.error({ err })`.

what do you think is the best way to handle this use case? should i instead capture the error before it reaches `pino-http` and use `log.fatal`?